packages/cli/src/callbacks.ts

```diff
@@ -193,6 +193,12 @@ export const getEnvName = async (currentName = 'env1'): Promise<string> => {
     message: 'Enter a name for the first environment in the workspace',
     name: currentName,
     default: currentName,
+    validate: (input: string): boolean | string => {
+      if (input === '') {
+        return 'Environment name cannot be empty'
+      }
+      return true
+    },
```

Comment:
style nit + typescript trick:
if we tell typescript that `const questions` is supposed to be of type `inquirer.Questions` it will be able to infer the types for the validate function, it will also enforce it better (if something is wrong the error will be on the definition of `questions` instead of on the call to `prompt`.
the style nit is to use the `?` operator for these types of functions that decide between two options (this is the more common practice in our code)
```suggestion
    validate: input => input === '' ? 'Environment name cannot be empty' : true,
```
after changing the definition to:
```typescript
const questions: inquirer.Questions = [{
```